scripts/alternat.sh

```diff
@@ -9,10 +9,29 @@ shopt -s expand_aliases
 
 panic() {
   [ -n "$1" ] && echo "$1"
-  echo "HA NAT setup failed"
+  echo "alterNAT setup failed"
   exit 1
 }
 
+load_config() {
+   if [ -f "$CONFIG_FILE" ]; then
+      . "$CONFIG_FILE"
+   else
+      panic "Config file $CONFIG_FILE not found"
+   fi
+   validate_var "eip_allocation_ids_csv" "$eip_allocation_ids_csv"
+   validate_var "route_table_ids_csv" "$route_table_ids_csv"
+}
+
+validate_var() {
+   var_name="$1"
+   var_val="$2"
+   if [ ! "$2" ]; then
+      echo "Config var "$var_name" is unset"
```

Comment:
Did you want to escape the double quotes so they appear literally?

Reply:
Good catch. Fixed in 04ad961.